src/auth/scheme/index.ts

```diff
@@ -0,0 +1,21 @@
+import * as Hapi from 'hapi';
+import validateUser from './validate_user';
+
+export default {
+  name: 'twine-auth',
```

Comment:
This can all be moved inside the API plugin if you prefer, it all just needs to be configured before the routes. I was using the plugins as code organisation approach, but may have borrowed a little too much from the node modules approach.